ingestion/src/metadata/ingestion/source/database/domodatabase/metadata.py

```diff
@@ -135,20 +141,36 @@ def get_tables_name_and_type(self) -> Optional[Iterable[Tuple[str, str]]]:
             logger.warning(error)
             self.status.failed(schema_name, error, traceback.format_exc())
 
+    def get_owners(self, owner: Owner) -> Optional[EntityReference]:
+        try:
+            owner_details = self.domo_client.users_get(owner.id)
+            if owner_details.get("email"):
+                user = self.metadata.get_user_by_email(owner_details["email"])
```

Comment:
maybe we could use pydantic model for owner details as well?

Reply:
otherwise LGTM

Reply:
Yes will add that 👍